src/Slices/CompileReports/UI/Page.test.tsx

```diff
@@ -228,12 +231,17 @@ test("When using the status filter with the In Progress opiton then the compile
       { name: "FilterPicker" }
     )
   );
-  await userEvent.click(screen.getByRole("option", { name: "Status" }));
+  await userEvent.click(
+    screen.getByRole("option", { name: words("compileReports.columns.status") })
+  );
 
-  const input = screen.getByPlaceholderText("Select compile status...");
+  const input = screen.getByPlaceholderText(
+    words("compileReports.filters.status.placeholder")
+  );
   await userEvent.click(input);
 
   const option = await screen.findByRole("option", { name: "in progress" });
```

Comment:
Did you not extract the "in progress" because it isn't yet available in the "words" dict?

Reply:
The idea was to extract all hardcoded strings, that are displayed in the UI. "success", "in progress" are hardcoded in your test here, and are values that are displayed to the UI. Doesn't matter where the string comes from, you should do the assertion/check/search based on the 'words' dict, not a magic string :)